modules/parquet/src/parquetjs/utils/decoders.ts

```diff
@@ -105,6 +105,7 @@ export function decodePage(cursor: CursorBuffer, options: ParquetOptions): Parqu
       page = decodeDataPageV2(cursor, pageHeader, options);
       break;
     case 'DICTIONARY_PAGE':
+    case 'RLE_DICTIONARY':
```

Comment:
Aren't these different enums (pageType vs Encoding)? They should not be assignable to the same value...

Reply:
Yes we don't need it here. Removed.